src/Discord.Net.Commands/CommandService.cs

```diff
@@ -77,6 +78,15 @@ public class CommandService : IDisposable
         /// </summary>
         public ILookup<Type, TypeReader> TypeReaders => _typeReaders.SelectMany(x => x.Value.Select(y => new { y.Key, y.Value })).ToLookup(x => x.Key, x => x.Value);
 
+        /// <summary>
+        ///     Represents all entity type reader <see cref="Type" />s loaded within <see cref="CommandService"/>.
+        /// </summary>
+        /// <returns>
+        ///     A <see cref="ILookup{TKey, TElement}"/> that the key is the object type to be read by the <see cref="TypeReader"/>
```

Comment:
Sounds kinda wordy and awkward. `A ILookup<TKey,TElement>; the key is the object type to be read by the TypeReader, while the element is the type of the TypeReader generic definition.` seems slightly better.